src/components/CytoscapeGraph/EmptyGraphLayout.tsx

```diff
@@ -19,6 +21,23 @@ type EmptyGraphLayoutState = {};
 
 export default class EmptyGraphLayout extends React.Component<EmptyGraphLayoutProps, EmptyGraphLayoutState> {
   render() {
+    if (this.props.isError) {
+      return (
+        <EmptyState className={emptyStateStyle}>
+          <EmptyStateTitle>Error loading Service Graph</EmptyStateTitle>
+          <EmptyStateInfo>
+            Service Graph cannot be loaded. Please access to the Message Center for more details.
```

Comment:
I'm not sure if "Message Center" is a name that the user knows about. Maybe just "Service Graph cannot be loaded" is enough, and we can assume the user will see the red alert icon in message center?

Reply:
Or even better, can we show a link "click here for details" that opens the message center?

Reply:
Well, Message Center is how the component is showed:
![image](https://user-images.githubusercontent.com/1662329/41396814-63bc1bea-6fb2-11e8-8632-9f623c7d831e.png)

But I think is good idea to add a link to open it, I'm going to add it.

Reply:
@jotak changed, let me know how it looks.